saleor/graphql/translations/types.py

```diff
@@ -124,12 +124,7 @@ class Meta:
 
     @staticmethod
     def resolve_product_variant(root: product_models.ProductVariant, info):
-        visible_products = product_models.Product.objects.visible_to_user(
-            info.context.user
-        ).values_list("pk", flat=True)
-        return product_models.ProductVariant.objects.filter(
-            product__id__in=visible_products, pk=root.id
-        ).first()
+        return product_models.ProductVariant.objects.filter(pk=root.id).first()
```

Comment:
If `root` is `ProductVariant` why we use filter and first here? Can't we just return root instance?